### PR TITLE
Add 10.11 release notes

### PR DESCRIPTION
**Pull request by tas50**
#### Description

Add 10.11 release notes

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [x] New content
- [ ] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [x] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

File: releases/2024-04-09-mondoo-10.11-is-out.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+```bash
+az acr login -n my_registry
+cnspec scan container registry my_registry.azurecr.io
+```
+
+## 🧹 IMPROVEMENTS
+
+### Collect logged in users in query packs
+
+Windows, Linux, and macOS query backs now collect the currently logged in users so you can understand active users on endpoints or remote connections to servers.
```
Comment (vjeffrey) [inline releases/2024-04-09-mondoo-10.11-is-out.md:33]: nit: backs -> packs

File: releases/2024-04-09-mondoo-10.11-is-out.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## 🥳 Mondoo 10.11 is out! This release includes Azure ACR scanning, expanded OS query packs, and more!
+
+Get this release: [Installation Docs](https://mondoo.com/docs/cnspec/) | [Package Downloads](https://releases.mondoo.com/cnspec/) | [Docker Container](https://hub.docker.com/r/mondoo/cnspec)
+
+---
+
+## 🎉 NEW FEATURES
+
+### Azure ACR Scanning
```
Comment (misterpantz) [inline releases/2024-04-09-mondoo-10.11-is-out.md:18]:
```suggestion
### Azure ACR scanning
```

File: releases/2024-04-09-mondoo-10.11-is-out.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+To login and scan a complete registry run:
+
+```bash
+az acr login -n my_registry
+cnspec scan container registry my_registry.azurecr.io
+```
+
+## 🧹 IMPROVEMENTS
+
+### Collect logged in users in query packs
```
Comment (misterpantz) [inline releases/2024-04-09-mondoo-10.11-is-out.md:31]:
```suggestion
### Collect logged-in users in query packs
```

File: releases/2024-04-09-mondoo-10.11-is-out.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+Windows, Linux, and macOS query packs now collect the currently logged in users so you can understand active users on endpoints or remote connections to servers.
```
Comment (misterpantz) [inline releases/2024-04-09-mondoo-10.11-is-out.md:33]:
```suggestion
Windows, Linux, and macOS query packs now collect the currently logged-in users so you can understand active users on endpoints or remote connections to servers.
```

File: releases/2024-04-09-mondoo-10.11-is-out.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+## 🐛 BUG FIXES AND UPDATES
+
+- Fix a failure running the `users.all(sshkeys == empty)` query.
+- Don't panic when the scan play is set to 0.
+- Ignore deactivated users in the Slack policy's 2FA check. Thanks for this fix [@jaybrueder](https://github.com/jaybrueder/)
+- Improve the AWS IAM users multi-factor authentication (MFA) enabled check to only check users with a password set.
```
Comment (misterpantz) [inline releases/2024-04-09-mondoo-10.11-is-out.md:40]:
```suggestion
- Improve the AWS IAM user "MFA enabled" check to only check users with a set password.
```

File: releases/2024-04-09-mondoo-10.11-is-out.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+- Fix the discovery of gcp organizations and folders.
+- Improve the `scan gcp` help output.
+- Improve failure output when a CLI command could not be parsed in the gcp provider.
```
Comment (misterpantz) [inline releases/2024-04-09-mondoo-10.11-is-out.md:43]:
```suggestion
- Improve failure output when a CLI command can't be parsed in the GCP provider.
```

**Review by misterpantz — APPROVED** [review]

**Pull request merged**